### PR TITLE
feat(menu): make menu widget more compatible with encoder

### PR DESCRIPTION
**Pull request by HX2003**
### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Checkpoints
- [ ] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Run `code-format.py` from the `scripts` folder. [astyle](http://astyle.sourceforge.net/install.html) needs to be installed.
- [ ] Update the documentation

### CODE REVIEW

File: src/extra/widgets/menu/lv_menu.c [hunk 1]
```diff
@@ -352,8 +352,9 @@ void lv_menu_set_load_page_event(lv_obj_t * menu, lv_obj_t * obj, lv_obj_t * pag
 {
     LV_ASSERT_OBJ(menu, MY_CLASS);
 
-    /* Make the object clickable */
     lv_obj_add_flag(obj, LV_OBJ_FLAG_CLICKABLE);
+    lv_obj_clear_flag(obj, LV_OBJ_FLAG_SCROLLABLE);
```
Comment (kisvegabor) [inline src/extra/widgets/menu/lv_menu.c:356]: Why is it required?

**Review by kisvegabor — COMMENTED** [review]
Thank you, only a minor question.

File: src/extra/widgets/menu/lv_menu.c (hunk 1 above)
Comment (HX2003) [reply src/extra/widgets/menu/lv_menu.c:356]: Without it, entering using encoder will turn into edit mode.

Comment (kisvegabor) [reply src/extra/widgets/menu/lv_menu.c:356]: I thought the target of this function should be a button but buttons are not editable. Can you show a minimal example which demonstrates the issue?

**Review by kisvegabor — COMMENTED** [review]
Sorry, it seems I forgot the send my review. See my comment above.

File: src/extra/widgets/menu/lv_menu.c (hunk 1 above)
Comment (HX2003) [reply src/extra/widgets/menu/lv_menu.c:356]: Actually the target is _menu_cont_  (thats the way I did it in the examples) which is not a button by default.

**Comment by kisvegabor** [comment]
Thank you. It's still not crystal clear to me, but I'm sure you know it better than me and tested it. :slightly_smiling_face: 

Merging.

**Pull request merged**